packages/modeling/src/maths/utils/OrthonormalFormula.js

```diff
@@ -0,0 +1,82 @@
+import * as mat4 from '../mat4/index.js'
+
+import * as vec2 from '../vec2/index.js'
+import * as vec3 from '../vec3/index.js'
+
+/*
+ * Class that defines the formula for convertion to/from orthonomal basis vectors.
+ * @see https://www.kristakingmath.com/blog/orthonormal-basis-for-a-vector-set
+ */
+export class OrthonormalFormula {
+  /**
+   * Construct the standard basis formula from the given plane.
+   * @param {plane} the plane of which to convert vertices to/from the orthonormal basis
+   */
+  constructor (plane) {
+    // plane normal is one component
+    this.plane = plane
+    // orthogonal vector to plane normal is one component
+    const o = vec3.orthogonal(vec3.create(), plane)
```

Comment:
Can we use a more descriptive variable name than `o`? I thought `rightVector` was better. Also `o` is a particularly bad variable name because it looks like `0` at a glance.